core/src/main/java/org/apache/accumulo/core/client/PluginEnvironment.java

```diff
@@ -163,7 +163,7 @@ interface Configuration extends Iterable<Entry<String,String>> {
    * @param className Fully qualified name of the class.
    * @param base The expected super type of the class.
    */
-  <T> T instantiate(String className, Class<T> base) throws Exception;
+  <T> T instantiate(String className, Class<T> base) throws ReflectiveOperationException;
```

Comment:
Will this change cause any compatibility issues?  Thinking it will not, but not completely sure. Seems like its source compatible, not sure about binary.